ansible/roles/cartodb.postgresql/tasks/postgis.yml

```diff
@@ -4,8 +4,10 @@
    apt: name={{ item }} state=present
    with_items:
     - libxml2-dev
-    - postgresql-9.3-postgis-2.2
-    - postgresql-9.3-postgis-scripts
+    - liblwgeom-2.1.8
```

Comment:
Interesting. No I do not know. These packages are listed as dependencies on the cartodb docs. I'll reproduce and investigate a bit.